backends-velox/src/test/scala/io/glutenproject/execution/VeloxFunctionsValidateSuite.scala

```diff
@@ -343,4 +344,48 @@ class VeloxFunctionsValidateSuite extends WholeStageTransformerSuite {
         }
     }
   }
+
+  test("map literal - offload") {
+    def validateOffloadResult(sql: String): Unit = {
+      // allow constant folding
+      withSQLConf("spark.sql.optimizer.excludedRules" -> "") {
```

Comment:
I'm not familiar with this API, this seems to exclude all optimization rules?

Reply:
yes, the main idea is to make constant folding rule work